ansible/environments/mac/hosts

```diff
@@ -0,0 +1,33 @@
+; the first parameter in a host is the inventory_hostname which has to be
+; either an ip
+; or a resolvable hostname
+
+; used for local actions only
```

Comment:
Does `local` mean Ubuntu hosts?
With Docker for Mac, Ansible will deploy OW without any remote connection.
So I thought `ansible_connection` should be `local`.